python_modules/libraries/dagster-databricks/dagster_databricks/databricks.py

```diff
@@ -609,6 +610,11 @@ def submit_run(self, run_config: Mapping[str, Any], task: Mapping[str, Any]) ->
                         {"pypi": {"package": f"{library_name}=={library.__version__}"}}
                     )
 
+            if "databricks-sdk" not in python_libraries:
+                libraries.append(
+                    {"pypi": {"package": f"databricks-sdk=={version('databricks-sdk')}"}}
```

Comment:
is this enough? given that 0.1.6 is installed and currently dagster-databricks requests <0.9 I think you should request a minimla version of 0.8 < 0.14 or at least >= 0.8, <0.9

Reply:
This will install whatever version is installed in the Dagster deployments' environment. The important part of solving this issue is that the databricks-sdk version needs to match between the client and the Databricks cluster. If `dagster-databricks` requires  <0.9, then <0.9 will be installed in the client, and this will ensure <0.9 is installed in the Databricks cluster. We shouldn't hardcode a version here - we should rely on installing whatever is installed locally (which should be whatever version `dagster-databricks` requests)